Doc/library/http.server.rst

```diff
@@ -392,7 +392,7 @@ provides three different variants:
       contents of the file are output. If the file's MIME type starts with
       ``text/`` the file is opened in text mode; otherwise binary mode is used.
 
-      For example usage, see the implementation of the :func:`test` function
+      For example usage, see the implementation of the :func:`~http.server.test` function
```

Comment:
This link won't work, because the `test()` function is (rightly) not documented. I think it would be more helpful to link to the source code, which can be done with the `:source:` directive.

Reply:
Thanks for the review, hopefully the latest version fixed this :)